lib/resources/r4/organization.yaml

```diff
@@ -0,0 +1,127 @@
+---
+name: Organization
+field_name_base_url: http://hl7.org/fhir/r4/organization-definitions.html#Organization
+fields:
+  - name: identifier
+    required: 'No'
+    cardinality: 0..*
+    type: Identifier
+    description: Identifies this organization across multiple systems.
+    url: http://hl7.org/fhir/R4/datatypes.html#Identifier
```

Comment:
Same here. All links should be secured ones.

Reply:
updated https://github.com/cerner/fhir.cerner.com/pull/326/commits/7e58be985de3b799900d13d575b7699ab10b9460